RTN-004.tex

```diff
@@ -53,7 +52,312 @@
 % ADD CONTENT HERE
 % You can also use the \input command to include several content files.
 
- 
+\section{Introduction to Data Preview 0}\label{sec:intro}
+
+There are three planned Data Previews (DPs): DP0 based on simulated data (2021), DP1 based on Commissioning Camera data, and DP2 based on Science Camera data.
+The Data Previews enable the community to access LSST-like data on a shared-risk basis and to participate in the science validation process. 
+
+\subsection{The Goals of DP0}\label{ssec:intro_goals}
+
+As described in \citeds{RTN-001}, the two goals of DP0 are:
+\begin{enumerate}
+\item To serve as an early integration test of the Data Management System (DMS) and inform further development of the science pipelines and the Rubin Science Platform (RSP).
+\item To prepare the community to use the LSST data products and services, and especially to enable early science with commissioning and year 1 data.
+\end{enumerate}
+
+In order to achieve these goals, community participation in DP0 has two main components:
+(1) performing scientific exercises with the data products and services in the RSP, and
+(2) taking action to inform or enhance the performance of the DMS, the RSP, or the science community.
+Participants from the science community, "DP0 delegates", will be data rights holders\footnote{Data rights holders are US or Chilean scientists, or individuals associated with an International Contribution (in-kind) proposal in progress, as defined in \citeds{RDO-013}.} who represent a diversity of backgrounds.
+The term "delegates" reflects the fact that these individuals will have the important role of representing the science community as learners and as contributors to the overall success of DP0.
+The nature of these two goal-driven components of DP0 participation will vary according to individual delegates' interests and experience.
+
+\subsection{Two Stages: DP0.1 and DP0.2}\label{ssec:intro_stages}
+
+The simulated data set for DP0 was generated by the Dark Energy Science Collaboration (DESC) and is being used under an agreement between DESC and Rubin.
```

Comment:
I think that it is "the DESC", i.e an agreement between the DESC and Rubin. Check with Phil.

Reply:
I made sure all appearances of "DESC" are "the DESC" (I think that was the only case were the "the" was missing).